sql/updates/world/3.3.5/2017_10_20_99_world_alchemy_spec.sql

```diff
@@ -0,0 +1,111 @@
+SET @TEXTID := xxx; -- Need 1
```

Comment:
11074

Reply:
Done, thanks. Just needs three empty gossip_menu_ids now.

Reply:
I tried, but cannot find anything.

Reply:
Is there a guideline for "custom" gossip_menu entries? Using entries > 40000 makes sense to me since sniffed ones for 3.3.5a don't seem to go any higher, just want to make sure.

Reply:
You can use 21402, 21403, 21404 if it appears impossible to find the correct IDs